src/anndata/_io/specs/methods.py

```diff
@@ -326,6 +326,30 @@ def write_raw(
     _writer.write_elem(g, "varm", dict(raw.varm), dataset_kwargs=dataset_kwargs)
 
 
+########
+# Null #
+########
+
+
+@_REGISTRY.register_read(H5Array, IOSpec("null", "0.1.0"))
+@_REGISTRY.register_read(ZarrArray, IOSpec("null", "0.1.0"))
+def read_null(_elem, _reader) -> None:
+    return None
+
+
+@_REGISTRY.register_write(H5Group, type(None), IOSpec("null", "0.1.0"))
+def write_null_h5py(f, k, _v, _writer, dataset_kwargs=MappingProxyType({})):
+    f.create_dataset(k, data=h5py.Empty("f"), **dataset_kwargs)
```

Comment:
I see in https://github.com/zarr-developers/zarr-specs/issues/318 you mention that `h5py.Empty` has no dtype but:

```python
>>> import h5py
>>> h5py.Empty("f")
Empty(dtype=dtype('float32'))
```
Am I missing something?

Reply:
This is inconsequential as far as I can tell but just wanted to make sure I understand.  It could also be relevant for the discussion